transhumandesign/kag-base
# [fixed] localhost building

## Status

- **READY**

## Description

* Fixes a bug related to building two structures at the same time on localhost.
* Fixes a bug where pages wouldn't change in localhost.
* Improves mod support by removing hack code.

## Steps to Test or Reproduce

On localhost, build a building.
If using the old build: you will notice that the building is made twice and one of the buildings will be destroyed.
If using the fixed build: no duplicate building is produced.

Netcode was altered so make sure to test on dedicated server. 

It should be noted that because important builder files were modified, this PR may break some mods- though it should be easy to update those mods to fix any issue.

resolves #2137, resolves #2081, resolves #2196
